website/docs/plugins/rq_launcher.md

```diff
@@ -0,0 +1,95 @@
+---
+id: rq_launcher
+title: RQ Launcher plugin
+sidebar_label: RQ Launcher plugin
+---
+[![PyPI](https://img.shields.io/pypi/v/hydra-rq-launcher)](https://pypi.org/project/hydra-rq-launcher/)
+![PyPI - License](https://img.shields.io/pypi/l/hydra-rq-launcher)
+![PyPI - Python Version](https://img.shields.io/pypi/pyversions/hydra-rq-launcher)
+[![PyPI - Downloads](https://img.shields.io/pypi/dm/hydra-rq-launcher.svg)](https://pypistats.org/packages/hydra-rq-launcher)
+[![Example application](https://img.shields.io/badge/-Example%20application-informational)](https://github.com/facebookresearch/hydra/tree/master/plugins/hydra_rq_launcher/example)
+[![Plugin source](https://img.shields.io/badge/-Plugin%20source-informational)](https://github.com/facebookresearch/hydra/tree/master/plugins/hydra_rq_launcher)
+
+The RQ Launcher plugin provides a launcher for distributed execution and job queuing based on [Redis Queue (RQ)](https://python-rq.org).
+
+### Installation
+This plugin requires Hydra 1.0 (Release candidate)
+```commandline
+$ pip install hydra-rq-launcher --pre
+```
+Usage of this plugin requires a [Redis server](https://redis.io/topics/quickstart).
+
+Note that RQ does [not support Windows](https://python-rq.org/docs/#limitations).
+
+### Usage
+Once installed, add `hydra/launcher=rq` to your command line. Alternatively, override `hydra/launcher` in your config:
+
+```yaml
+defaults:
+  - hydra/launcher: rq
+```
+
+The configuration packaged with the plugin is defined [here](https://github.com/facebookresearch/hydra/blob/master/plugins/hydra_rq_launcher/hydra_plugins/hydra_rq_launcher/config.py). The default configuratio is as follows:
+
+```yaml
+# @package hydra.launcher
+cls: hydra_plugins.hydra_rq_launcher.rq_launcher.RQLauncher
+params:
+  enqueue:
+    job_timeout: null                  # maximum runtime of the job before it's interrupted and marked as failed (in sec)
+    ttl: null                          # maximum queued time before the job before is discarded (in sec)
+    result_ttl: 864000                 # how long successful jobs and their results are kept (in sec), default: 10 days
+    failure_ttl: 8640000               # specifies how long failed jobs are kept (in sec), default: 100 days
+    at_front: false                    # place job at the front of the queue, instead of the back
+    job_id: null                       # job id, will be overidden automatically by a uuid unless specified explicitly
+    description: null                  # description, will be overidden automatically unless specified explicitly
+  queue: default                       # queue name
```

Comment:
I know adding comments is making it more useful, but:
1. the comments are in config.py.
2. it makes it harder to update, can't just copy-paste the output of the --cfg command,

For now let's keep this.